lib/IRGen/IRGen.cpp

```diff
@@ -148,6 +148,9 @@ swift::getIRTargetOptions(const IRGenOptions &Opts, ASTContext &Ctx) {
   if (Clang->getTargetInfo().getTriple().isOSBinFormatWasm())
     TargetOpts.ThreadModel = llvm::ThreadModel::Single;
 
+  if (Clang->getTargetInfo().getTriple().hasDefaultEmulatedTLS())
+    TargetOpts.EmulatedTLS = true;
```

Comment:
We probably should honour `-Xcc -femulated-tls` as well.

Reply:
I chose the established `-Xcc` flags to control this, because this only affects inline functions from C/C++ headers that have been spliced into Swift IR anyway, as Swift doesn't seem to have a TLS annotation for its own variables.